Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -61,15 +65,16 @@ public ActionResult WebhookPost([FromForm, ModelBinder(BinderType = typeof(PlexB
                     break;
             }
 
-            return Ok(); //doesn't need to be an ApiStatus.OK() since really, all I take is plex.   
+            return Ok(); //doesn't need to be an ApiStatus.OK() since really, all I take is plex.
         }
+
         #region Plex events
 
         [NonAction]
-        private static void TraktScrobble(PlexEvent evt, ScrobblePlayingStatus type)
+        private void TraktScrobble(PlexEvent evt, ScrobblePlayingStatus type)
```

Comment:
Could have stayed static.

Reply:
Same reason as above — because of the logger.